pyopensprinkler/__init__.py

```diff
@@ -161,26 +174,37 @@ def refresh(self):
                 self._stations[i] = Station(self, i)
 
     def _refresh_state(self):
-        try:
-            (_, content) = self.request("/ja")
-        except OpenSprinklerApiError as exc:
-            (_, err_code) = exc.args
-            if err_code == 32:
-                # Backwards compatibility for pre 2.1.6
-                (_, settings) = self.request("/jc")
-                (_, options) = self.request("/jo")
-                (_, stations) = self.request("/jn")
-                (_, status) = self.request("/js")
-                (_, programs) = self.request("/jp")
-                content = {
-                    "settings": settings,
-                    "options": options,
-                    "stations": stations,
-                    "status": status,
-                    "programs": programs,
-                }
-            else:
-                raise exc
+        use_ja = True
```

Comment:
I think here we can check for state if not none to check for firmware version to default to False if the firmware < 216 so it does try /ja except for the first time

Reply:
It does this behavior already actually.